src/components/RequestInviteForm.js

```diff
@@ -0,0 +1,127 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { auth, db } from '../firebaseInit';
+import { getRequestInviteOperation } from '../operations';
+import { postOperation, fetchMemberIfNeeded } from '../actions';
+import YoutubeVideo from './YoutubeVideo';
+
+interface Props {
+  partialAuthData: Map<string, string>;
+  toMid: string;
+}
+
+const getNumberSuffix = (len: number) => {
+    const exclusiveMax = Math.pow(10, len);
+    return (Math.floor(Math.random() * exclusiveMax) + exclusiveMax).toString().substring(1);
+}
+
+const getUserName = (fullName: string) => {
```

Comment:
@markulrich feel like these should be in another module - where do you think go best?

Reply:
Maybe `operations.js`? Or could make a new file. Can we call this something like `getMemberId`?

Reply:
renamed function, and moved into a new file tentatively called `member.js`